_rules/html-has-lang-b5c3f8.md

```diff
@@ -56,88 +53,32 @@ The `lang` attribute specified has a non-empty value.
 <html lang="en"></html>
 ```
 
-#### Passed Example 2
-
-The `xml:lang` attribute specified has a non-empty value.
-
-```html
-<html xml:lang="en"></html>
-```
-
-#### Passed Example 3
-
-The `lang` and `xml:lang` attributes specified have a non-empty value.
-
-```html
-<html xml:lang="en" lang="en"></html>
-```
-
-#### Passed Example 4
-
-The `lang` attribute specified has a non-empty value. The rule expects a non-empty value on either the `lang` or `xml:lang` attributes.
-
-```html
-<html xml:lang="" lang="en"></html>
-```
-
-#### Passed Example 5
-
-The `xml:lang` attribute specified has a non-empty value. The rule expects a non-empty value on either the `lang` or `xml:lang` attributes.
-
-```html
-<html xml:lang="en" lang=""></html>
-```
-
 ### Failed
 
 #### Failed Example 1
 
-There were no `lang` or `xml:lang` attribute specified.
+There were no `lang` attribute specified.
 
 ```html
 <html></html>
 ```
 
 #### Failed Example 2
 
-The `xml:lang` attribute specified is empty ("").
-
-```html
-<html xml:lang=""></html>
-```
-
-#### Failed Example 3
-
 The `lang` attribute specified is empty ("").
 
 ```html
 <html lang=""></html>
 ```
 
-#### Failed Example 4
-
-The `lang` and `xml:lang` attributes specified are empty ("").
-
-```html
-<html xml:lang="" lang=""></html>
-```
-
-#### Failed Example 5
+#### Failed Example 3
 
-The `lang` attribute consists of only [whitespace](#whitespace).
+The `lang` attribute consists of only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).
 
 ```html
 <html lang=" "></html>
 ```
 
-#### Failed Example 6
-
-The `xml:lang` attribute consists of only [whitespace](#whitespace).
-
-```html
-<html xml:lang=" "></html>
-```
-
 ### Inapplicable
 
 #### Inapplicable Example 1
```

Comment:
Might be worth adding an Inapplicable Example for MathML as well - there's no `lang` attribute in MathML but `xml:lang` is allowed
https://www.w3.org/TR/MathML3/mathml.html#appendixi_index.att